app/containers/Orders/selectors.js

```diff
@@ -0,0 +1,18 @@
+import { createSelector } from 'reselect';
+
+const getOrders = (state) => state.get('orders').toJS();
+
+const makeGetOrders = () => createSelector(getOrders, (orders) => orders);
```

Comment:
you can use the return of the function already defined `getOrders`, one of the advantages of `reselect` is that you can compose your functions..

see the examples in the other selectors =)

Reply:
Fine. I will check the examples